tests/main/views/test_brief_response.py

```diff
@@ -252,6 +261,20 @@ def test_cannot_create_brief_response_when_supplier_isnt_eligible(self, live_dos
         assert res.status_code == 400
         assert 'Supplier is not eligible to apply to this brief' in res.get_data(as_text=True)
 
+    def test_cannot_create_a_brief_response_if_framework_status_is_not_live_or_expired(self, expired_dos_framework):
+        for framework_status in ['coming', 'open', 'pending', 'standstill']:
+            with self.app.app_context():
+                db.session.execute(
+                    "UPDATE frameworks SET status=:status WHERE slug='digital-outcomes-and-specialists'",
```

Comment:
Had a chat with Ben, gonna go down the route of targeting the fixture and updating it. It'll reset itself every time it's used by a test so no issues!